src/components/Sprint.vue

```diff
@@ -67,10 +72,29 @@
         const sprintId = this.sprint.id;
         
         FirebaseService.removeSprint(projectId, sprintId);
-        store.commit('removeSprintFromProject', {
+        this.$store.commit('removeSprintFromProject', {
           projectId,
           sprintId
         });
+      },
+      setUpdateModel() {
+        this.updatedSprint = {
+          startDate: this.sprint.startDate,
+          endDate: this.sprint.endDate
+        }
+        return this.editMode = true;
+      },
+      updateSprint() {
+        this.updatedSprint = {
+          ...this.sprint,
+          startDate: this.updatedSprint.startDate,
+          endDate: this.updatedSprint.endDate
+        };
+
+        FirebaseService.saveSprint(this.project.id, this.updatedSprint);
+        this.$store.commit('addSprint', { projectId: this.project.id, sprint: this.updatedSprint });
```

Comment:
What do you think about moving all the $store.commit stuff to `firebase-service.js`? That way it's only in one place and we aren't repeating ourselves if we ever want to save a sprint/team/project in more than one place.

Reply:
The thought crossed my mind but I was on a roll haha. I agree it is the more ideal way of doing it.

If I get time I will go through and move them all, but not in this PR.